packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/android/remotesignout.md

```diff
@@ -2,25 +2,65 @@
 
 Signing out of Okta requires the app to open a browser and navigate to the [end session endpoint](https://developer.okta.com/docs/reference/api/oidc/#logout). Okta destroys the user's session and immediately redirects back to your application.
 
-To do this, you must define a callback route for the sign-out process. 
-
-`TODO: Instructions to add signout redirect URI to gradle file`
-
-Next, add it as an allowed **Logout redirect URI** in the Okta Developer Console:
+To do this, you must define a callback route for the sign-out process.
+This should of been done in the previous step: [Create an Okta application](https://developer.okta.com/docs/guides/sign-into-mobile-app/android/create-okta-application/)
+but if you haven't done it:
 
 <a href="https://login.okta.com/" target="_blank" class="Button--blue">Go to Console</a>
 
 1. Select **Applications**, and then pick your application.
 2. Select **General** and click **Edit**.
-3. In the **Logout redirect URI**, enter the callback that you defined. For example, `com.okta.example:/signoutCallback`
+3. In the **Logout redirect URI**, enter the callback that you defined. For example, `com.okta.example:/logout`
 4. Click **Save**.
 
-`TODO: Is the rest handled by our OIDC SDK? Does the developer need to hook anything up in their app?`
+Make sure the `endSessionRedirectUri` in `OIDCConfig` is the same as the one defined in **Logout redirect URI**. For example if creating the `OIDCConfig` by builder:
```

Comment:
Make sure that the `endSessionRedirectUri` in `OIDCConfig` is the same as the one defined in the **Logout redirect URI** section. For example, if you are creating the `OIDCConfig` by builder: